connector_jira/models/res_users/common.py

```diff
@@ -69,10 +69,12 @@ def link_with_jira(self, backends=None, raise_if_mismatch=False):
                         if raise_if_mismatch:
                             raise exceptions.UserError(
                                 _(
-                                    'Several users found with "%s" set to "%s". '
-                                    "Set it manually."
+                                    'Several users found with "%(resolve_by_key)s"\
+                                        set to "%(resolve_by_value)s". '
```

Comment:
You do not need the break line here, just do as much string concatenation as you want.
Not a blocker

Reply:
Thanks for review, branch updated